src/main/java/ru/mtsbank/fintech/ScheduledTasks.java

```diff
@@ -25,19 +25,16 @@ public ScheduledTasks(AnimalRepositoryImpl animalRepository) {
     @Scheduled(fixedRate = 1000 * 60) // 1 минута
     public void animalScheduledLog() {
         log.info("FindLeapYearNames animal {}", animalRepository.findLeapYearNames());
-
         log.info("FindDuplicate animal {}", animalRepository.findDuplicate());
-
         log.info("FindAverageAge {}", animalRepository.findAverageAge(animalRepository.getAnimalArray().get("CAT")));
 
-        log.info("FindOldAndExpensive {}", animalRepository.findOldAndExpensive(5, animalRepository.getAnimalArray().get("CAT")));
         try {
             log.info("FindOlder animal {}", animalRepository.findOlderAnimal(10).toString());
             log.info("FindMinConstAnimals {}", animalRepository.findMinConstAnimals(animalRepository.getAnimalArray().get("CAT"), 3));
-            log.info("FindOldAndExpensive {}", animalRepository.findOldAndExpensive(5,animalRepository.getAnimalArray().get("CAT")));
+            log.info("FindOldAndExpensive {}", animalRepository.findOldAndExpensive(5, animalRepository.getAnimalArray().get("CAT")));
 
         } catch (Exception e) {
-            System.out.println("Exception! : " + e);
+            log.info("Exception! : " + e);
```

Comment:
У логгера есть несколько уровней логирования.
info - для информационных сообщений.
error - для ошибок.
debug & trace - соответственно.

Еще, лучше разделить сообщение и стектрейс

```suggestion
           log.error("Exception! : {}", e.getMessage(), e);
```